src/App.tsx

```diff
@@ -1,52 +1,59 @@
+import {
+  Routes,
+  Route,
+  Navigate,
+} from 'react-router-dom';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
-
-// const tabs = [
-//   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-//   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-//   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-// ];
+import { NavLinkItem } from './components/NavLinkItem';
+import { HomePage } from './pages/HomePage';
+import { PageNotFound } from './pages/PageNotFound';
+import { TabsPage } from './pages/TabsPage';
 
 export const App = () => (
   <>
-    {/* Also requires <html class="has-navbar-fixed-top"> */}
     <nav
```

Comment:
Put nav to a Navigation component and create a Layout component that will serve as a wrapper for all your routes. Within this Layout, we'll include a Navigation component and a section to wrap the content. To achieve this, we can use the Outlet component from react-router-dom.

```
<Routes>
  <Route path="/" element={<Layout />}>
    .. homepage
    ...tabs
    ...
 </Routes>   
```